app/models/miq_report_result.rb

```diff
@@ -53,6 +53,20 @@ def status_message
     miq_task.nil? ? _("The task associated with this report is no longer available") : miq_task.message
   end
 
+  # Use this method over `report_results` if you don't plan on using the
+  # `binary_blob` associated with the MiqReportResult (chances are you don't
+  # need it).
+  def has_report?
```

Comment:
Honestly, I was thinking about this and want to make this "more verbose".  `report?`, and even `has_report?` gives the impression that `report` is a relation, not a column on this report.  I am almost thinking of renaming this to `valid_report_column?` or something like that.

Thoughts?

Reply:
hmm, good point... why would report not be a `MiqReport`?  It seems like the report result might be useless without a report object... but I'm not sure how it's used if there isn't a report.  Maybe `linked_report?` since the UI use case you changed is to change the selected node in the chargeback tree for the report corresponding to this report result.

Reply:
Again, this is a serialized report shoved into the `report` column, so it could be serialized to anything technically speaking.

`linked_report?` to me still sounds like we are talking about a relation, when in fact, this is part of the record itself.  Does that rational make sense?